@xen-orchestra/lite/src/i18n.ts

```diff
@@ -0,0 +1,110 @@
+import { createI18n } from "vue-i18n";
+import en from "@/locales/en.json";
+import fr from "@/locales/fr.json";
+
+export default createI18n<[typeof en], "en" | "fr">({
+  locale: "fr",
```

Comment:
`locale` is used to set the default language I guess?
Wouldn't it be better to set the default language to English?
```suggestion
  locale: "en",
```